contracts/VaultV3.vy

```diff
@@ -1338,6 +1383,27 @@ def _process_report(strategy: address) -> (uint256, uint256):
     return (gain, loss)
 
 # SETTERS #
+
+@external
+def setName(name: String[64]):
+    """
+    @notice Change the vault name.
+    @dev Can only be called by the Role Manager.
+    @param name The new name for the vault.
+    """
+    assert msg.sender == self.role_manager, "not allowed"
+    self.name = name
+
+@external
+def setSymbol(symbol: String[32]):
```

Comment:
It may break some setups, but should be easy to fix.

It is the same functionality as the v2 vaults have and has been used in the past so i would assume people have a way to deal with it.